cmd/branchingconfigmanagers/tide-config-manager/main_test.go

```diff
@@ -155,6 +160,83 @@ func TestReconcile(t *testing.T) {
     - openshift-4.9
     repos:
     - openshift/dummy
+`},
+		},
+		{
+			name: "bugzilla/valid-bug will be removed during code freeze from repo with main",
+			args: args{
+				target: afero.NewMemMapFs(),
+				event:  codeFreezeEvent,
+				config: &prowconfig.ProwConfig{Tide: config.Tide{Queries: config.TideQueries{
+					{
+						Repos:            []string{"openshift/dummy"},
+						Labels:           []string{"bugzilla/valid-bug"},
+						IncludedBranches: []string{"main", "release-4.9"},
+					},
+				}}},
+			},
+			wantErr: false,
+			expectedShardFiles: map[string]string{
+				"openshift/dummy/_prowconfig.yaml": `tide:
```

Comment:
Please check https://github.com/openshift/ci-tools/blob/master/pkg/testhelper/testhelper.go#L87

Reply:
Sounds good. You can do it on a follow-up